packages/pwa-storefront-mcp/src/utils/utils.js

```diff
@@ -37,37 +32,43 @@ export const toPascalCase = (str) =>
     str.replace(/(^\w|[-_\s]\w)/g, (match) => match.replace(/[-_\s]/, '').toUpperCase())
 
 /**
- * Spawns a pseudo-terminal process and captures its output.
- *
- * This is useful for running CLI tools that emit different output
- * when run in a TTY environment (e.g., with color, formatting, or paging).
+ * Runs a shell command and captures its stdout/stderr as a string.
```

Comment:
the 3rd party npm package that I was using to communicate with the create-app script was old and didn't support newer versions of node. So I decided to remove it and use vanilla apis. I can do that because I found the root of the problem for truncated output in the create-app script itself.